src/sentry/api/endpoints/organization_index.py

```diff
@@ -28,17 +28,22 @@
 )
 from sentry.services.organization.provisioning import organization_provisioning_service
 from sentry.signals import org_setup_complete, terms_accepted
+from sentry.types.region import get_local_region
 
 
 class OrganizationPostSerializer(BaseOrganizationSerializer):
     defaultTeam = serializers.BooleanField(required=False)
     agreeTerms = serializers.BooleanField(required=True)
+    region = serializers.CharField(required=True)
     idempotencyKey = serializers.CharField(max_length=IDEMPOTENCY_KEY_LENGTH, required=False)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if not (settings.TERMS_URL and settings.PRIVACY_URL):
             del self.fields["agreeTerms"]
+
+        if not options.get("hybrid_cloud.multi-region-selector"):
+            del self.fields["region"]
```

Comment:
This is a bit of a hack, but we need this for consistent UI validation. Otherwise we'll need to hack in some functionality replicating the error message that our error message parsing handles automatically.

This might also break API requests going forward when this option is enabled so this may need to be rethought 🤔